test/built-ins/Proxy/enumerate/removed-does-not-trigger.js

```diff
@@ -0,0 +1,48 @@
+/* global assert */
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-proxy-object-internal-methods-and-internal-slots
+description: >
+  Enumerate trap was removed and it should not be triggered anymore.
+includes: [compareArray.js]
+features: [Symbol.iterator]
+---*/
+
+var x;
+var target = [1, 2, 3];
+var p = new Proxy(target, {
+  enumerate: function() {
+    throw new Test262Error(
+      "An enumerate property on handler object shouldn't trigger a Proxy trap"
+    );
+  }
+});
+
+var forInResults = [];
+for (x in p) {
+  forInResults.push(x);
+}
+
+assert(compareArray(forInResults, [0, 1, 2]));
+
+var forOfResults = [];
+for (x of p) {
+  forOfResults.push(x);
+}
+
+assert(compareArray(forInResults, [1, 2, 3]));
```

Comment:
These should be `string` too. Also, we should compare `forOfResults` instead of `forInResults`.